nx-X11/programs/Xserver/hw/nxagent/Screen.c

```diff
@@ -4179,6 +4449,7 @@ int nxagentAdjustRandRXinerama(ScreenPtr pScreen)
 	     refcnt by 1. We decrease it again by calling only
 	     RRModeDestroy() and forget about prevmode */
 	  RRModeDestroy(mymode);
+	  RRCrtcSet(pScrPriv->crtcs[i], mymode, new_x, new_y, RR_Rotate_0, 1, &(pScrPriv->outputs[i]));
```

Comment:
this is the line I suggested to add, is this still required?

When I started all this I hade code cascades like this, too. But in the end it all lead to basically the same behaviour. So I _feel_ that your change being too big. From my POV it is simply a matter of the position not changing in case of a resize. That probably is the only thing that needs to be reconsidered. But I may be totally wrong, I have to dig deeper...